lib/src/conflicts.rs

```diff
@@ -269,9 +270,9 @@ pub fn materialize_merge_result(
                         // The vast majority of conflicts one actually tries to
                         // resolve manually have 1 base.
                         let base_str = if hunk.removes().len() == 1 {
-                            "base".to_string()
+                            Cow::Borrowed("base")
                         } else {
-                            format!("base #{}", base_index + 1)
+                            Cow::Owned(format!("base #{}", base_index + 1))
```

Comment:
This feels like a premature optimization to me. Does it make a measurable difference?

Reply:
No, I don't think it makes a big difference, so I'll remove it.